docs/figure_file_format.rst

```diff
@@ -0,0 +1,232 @@
+
+OMERO.figure files
+==================
+
+OMERO.figure files are JSON data, stored in OMERO as OriginalFiles linked to FileAnnotations.
+
+File Annotations
+----------------
+
+The FileAnnotations are identified as OMERO.figure files using a namespace of ``omero.web.figure.json``.
```

Comment:
using the namespace ``omero.web.figure.json``
using namespace of sounds a bit strange to me